community/events/sprints.md

```diff
@@ -0,0 +1,264 @@
+
+3. **Update the Status on the Project Board**: Once the issue has been added to
+   the project board (it normally takes our CI workflow 30 seconds to a minute),
+   update the status of the issue on the project board based on the type of skill
+   needed.
+
+
+The current types of tasks in our board include:
+
+    * Python packaging
+    * Beginner friendly / non technical
+    * Dev Ops / GitHub actions (Continuous Integration)
+    * Python programming
+    * Website - css or ruby
+
+:::{todo}
+we should add content review and sphinx as a status options for people that review our guidebook, run through tutorials etc.
+:::
+
+:::{important}
+If you are creating a new `help-wanted` issue it's critical that you include as much detailed information in the issue as possible. Imagine someone else reading the issue (that is not you!). In reading the issue, does an outside contributor have:
+
+* clearly understand the problem? (be specific, provide examples, links etc)
+* have enough information to solve the problem?
+* know exactly where the problem is occurring (provide links if you can or code examples )
+* have enough information needed to solve the problem?
+
+When in doubt, more information is always better!
+:::
+
+Labeling issues with `help-wanted` / `sprintable` throughout the year as they are opened will save significant preparation time when a Sprint event arises. It will also make it easier for fly-by contributors to find things to help us with throughout the year.
+
+### Pre-Sprint Event Tasks
+
+#### Triage (Help-Wanted) Issues Across the pyOpenSci Organization
+
+Before a sprint begins, someone on the pyOpenSci team should go through and
+triage all of the open issues in the organization to determine:
+
+* Whether some of them are sprintable.
+* Whether they have enough specific and detailed information for someone to work
+  on the issue without too much help during a sprint.
+
+This could be a small team exercise at a pre-meeting event held online for
+pyOpenSci staff and sprint community members (if there are any contributors
+participating).
+
+The information found in the body of an issue is critical to running an effective
+sprint. If the issue has very specific information that gives a potential
+contributor everything they need to know to begin working, they will have fewer
+questions during the sprint. If you have a table of 10-20 people sprinting for
+pyOpenSci, this means the person running the sprint will have less to do on-site!
+Sprints are busy—please add as much specific information as you can to each issue!
+
+#### THIS LIKELY WON'T WORK Ensure issue and pull request templates are up to date
+
+A challenge of a successful sprint is that there will be many issues and pull requests to triage after the sprint. To keep track of new issues and pr's, during an event, every pyOpenSci GitHub repository should have a sprint issue and pull request template. Adding this template only needs to be completed once, however
```

Comment:
I think we should go into scipy with a manual process and just see if automating adding to the board would help. i suspect it won't be that much more beneficial and more infra to maintain. If the pr template worked then it would make sense. but because i don't think the pr template will work (i remember fighting with this for stravalib) then manual labeling might be our best bet here. We could however have an issue template for sprints. that may or may not be useful .

Reply:
if we collet github usernames at the sprint start, the remote person would know who is contributing by their username making it even easier to identify sprint related activity.

Reply:
@kierisi i see my THIS WONT WORK comment now 😆 it was not visible to me wednesday. but i do remember writing it now.